tools/nuget.config

```diff
@@ -4,6 +4,7 @@
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="system.commandline" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json" />
+    <!--<add key="nugettest.org" value="https://apiint.nugettest.org/v3/index.json" />-->
```

Comment:
Commented out code

Reply:
This is intentional for convenience, as we sometimes use the test feed.